api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/FaltososLevantamentoARVDataSet.java

```diff
@@ -188,25 +191,46 @@ public DataSetDefinition constructDataSet() {
 
   private List<ColumnParameters> getColumnsForChildren() {
     ColumnParameters lessThan15 =
-        new ColumnParameters("under15", "under 15 year ", "age=<15", "lessThan15");
+        new ColumnParameters(
+            "under15",
+            "under 15 year ",
+            DimensionKeyForAge.bellow15Years.getDimension(),
+            "lessThan15");
     ColumnParameters lessTha15Female =
         new ColumnParameters(
-            "under15Female", "under 15 years Female", "gender=F|age=<15", "lessThan15Female");
+            "under15Female",
+            "under 15 years Female",
+            DimensionKeyForGender.female.and(DimensionKeyForAge.bellow15Years).getDimension(),
+            "lessThan15Female");
     ColumnParameters lessThan15Male =
         new ColumnParameters(
-            "under15Male", "under 15 years Male", "gender=M|age=<15", "lessThan15Male");
+            "under15Male",
+            "under 15 years Male",
+            DimensionKeyForGender.male.and(DimensionKeyForAge.bellow15Years).getDimension(),
+            "lessThan15Male");
 
     return Arrays.asList(lessThan15, lessTha15Female, lessThan15Male);
   }
 
   private List<ColumnParameters> getColumnsForAdult() {
     ColumnParameters greaterThan15 =
-        new ColumnParameters("above15", "above 15", "age=15+", "greaterThan15");
+        new ColumnParameters(
+            "Equal or above 15",
```

Comment:
a name should be camel case but NOT include spaces, only the labels can have the spaces

Reply:
Ok. I'll change

Reply:
done